src/utils/normalize-mime-parts.ts

```diff
@@ -73,12 +73,17 @@ export function normalizeMimeParts(
 	message: { [key: string]: any },
 	{ origin, jwtToken }: { jwtToken?: string; origin?: string }
 ) {
-	const processAttachment = ({ ...attachment }) => {
+	const processAttachment = ({ ...attachment }, type: string) => {
```

Comment:
Do we need a second argument? I thought that `attachment` would have a property for `contentType` without any other arguments.

Reply:
Oh. That actually, I added just not to write same `normalizeType(attachment.contentType)` code. But, removed now.